### DISCUSSION TITLE
Help us name the 1.3 release!

### DISCUSSION THREAD

**Discussion by tacastillo** [dagster-io/dagster#13478] [Community]
# Update

**We have a release name!!** Thanks, @linssendane, for suggesting Smooth Operator!
<img width="929" alt="image" src="https://user-images.githubusercontent.com/10715711/233224168-725a3ee8-77ce-4d1a-98ae-d7c55cb96b9c.png">

Hey folks! Tim from the Dagster team here.

We'll be releasing Dagster v1.3 soon and want your feedback!

Every major and minor release gets named after a hit single or an album from a pop diva. Sometimes, the name of the song/album is inspired by what changes are coming with the release.

For inspiration, 1.3 will have:
- Revised staleness ontology
- Customizing Asset Reconciliation
- Automatic source asset observations
- Pythonic Resources and Config
- Surfacing materializing partitions

And here are some songs/albums we've previously named our versions after!

- [Formation](https://www.youtube.com/watch?v=WDZJPJV__bQ) (Beyonce)
- [Thank U, Next](https://www.youtube.com/watch?v=gl1aHhXnN1k) (Ariana Grande)
- [Hello](https://www.youtube.com/watch?v=YQHsXMglC9A) (Adele)
- [Cool for the Summer](https://www.youtube.com/watch?v=il9nqWw9W3Y) (Demi Lovato)
- [Lucky Star](https://www.youtube.com/watch?v=ThHz9wlBeLU) (Madonna)
- [The Edge of Glory](https://www.youtube.com/watch?v=QeWBS0JBNzQ) (Lady Gaga)
- [Laundry Service](https://www.youtube.com/watch?v=7BPE5sR6oUU&list=PL035E8452800ACEA5) (Shakira)
- [In the Zone](https://www.youtube.com/watch?v=we8h_2RFAyA) (Britney Spears)
- [Waiting to Exhale](https://www.youtube.com/watch?v=_FT7ciAPuJk&list=PL_sN_rrP4od0gZ4C3LGkgcaRpVa_koQX4) (Whitney Houston, et al.)

Drop a comment with your suggestions, and upvote the suggestions you like!

![1 3](https://user-images.githubusercontent.com/10715711/230955828-e3a5edd7-b9af-41eb-9614-0ecd63dda9a0.png)

**Comment by alex-orlovskyi** [comment]
After looking through upcoming changes in v1.3 - first association in mind with [Laundry Service](https://www.youtube.com/watch?v=7BPE5sR6oUU&list=PL035E8452800ACEA5) ^^

**Comment by GH-maggio** [comment]
[Uptown Funk](https://www.youtube.com/watch?v=OPf0YbXqDm0) (Bruno Mars)

**Comment by linssendane** [comment]
[Smooth Operator](https://open.spotify.com/track/1Hv1VTm8zeOeybub15mA2R) - Sade

... since interacting with your workflows is about to get real smooth! 😁

**Comment by tacastillo** [comment]
All I Wanna Do

**Comment by tacastillo** [comment]
Control

**Comment by tacastillo** [comment]
Don't Start Now

**Comment by tacastillo** [comment]
Dressed to Kill

**Comment by tacastillo** [comment]
Genie in a Bottle

**Comment by tacastillo** [comment]
Havana

**Comment by tacastillo** [comment]
Ironic

**Comment by tacastillo** [comment]
Irreplaceable

**Comment by tacastillo** [comment]
New Rules

**Comment by tacastillo** [comment]
The Middle

**Comment by tacastillo** [comment]
What You Waiting For

**Comment by tacastillo** [comment]
You're Still the One

**Comment by ei-grad** [comment]
Titanium

**Comment by slopp** [comment]
Pythonic resources ➡️  Anaconda Nicki Minaj

**Comment by tacastillo** [comment]
<img width="929" alt="image" src="https://user-images.githubusercontent.com/10715711/233224216-c3cd8843-6797-4a81-a241-8222945818f4.png">
Dagster v1.3 will now be referred to as Smooth Operator!

**Comment by ipeterfulop** [comment]
Hurricane (Ofenbach, Ella Henderson)

**Comment by ipeterfulop** [comment]
Stormbringer (Deep Purple - v 2009 remastered)
